jinahub/rankers/MinRanker/tests/requirements.txt

```diff
@@ -0,0 +1,2 @@
+jina==2.0.10
```

Comment:
I think we should use `git+https://github.com/jina-ai/jina.git@v2.0.10#egg=jina` for the tests

Reply:
What is the reason for that?

Reply:
This does not use the pypi package. It instead uses git clone to download the package code.
The advantage is that our CI runs do not pollute the pypi download statistics.
See this Slack post by nan: https://jinaai.slack.com/archives/C0183GCN8AK/p1624870427247100